bookwyrm/static/css/bookwyrm.css

```diff
@@ -57,10 +49,6 @@ html {
     display: none;
 }
 
-.hidden {
-    display: none !important;
-}
-
 .hidden.transition-y,
 .hidden.transition-x {
```

Comment:
`is-hidden` already exists in Bulma. It has the same declarations as `hidden`. ;)

https://bulma.io/documentation/helpers/visibility-helpers/#other-visibility-helpers

Reply:
Yeah, I'm with you. This css class adds a small transition to elements on show/hide -- since they now receive `is-hidden` instead of `hidden` from the javascript, the declaration on lines 52 and 53 need to be `.is-hidden.transition-y, .is-hidden.transition-x`